tools/implib/implib.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net461</TargetFrameworks>
```

Comment:
Noticed this on a few projects. Do we not need a standard or core version here?

Reply:
There are few projects left off for now.  They are not needed for our current requirements.  I have noted about this in the pull request's description.